src/pubkey.cpp

```diff
@@ -388,7 +388,7 @@ ECCVerifyHandle::ECCVerifyHandle()
 {
     if (refcount == 0) {
         assert(secp256k1_context_verify == nullptr);
-        secp256k1_context_verify = secp256k1_context_create(SECP256K1_CONTEXT_VERIFY);
+        secp256k1_context_verify = secp256k1_context_create(SECP256K1_CONTEXT_NONE);
```

Comment:
We could equally use the static context here if we want to get rid of all the refcounting (but that's a somewhat larger change).